applications/FluidDynamicsApplication/python_scripts/python_solvers_wrapper_fluid.py

```diff
@@ -6,6 +6,12 @@ def CreateSolverByParameters(model, solver_settings, parallelism):
 
     solver_type = solver_settings["solver_type"].GetString()
 
+    if solver_type == "ale_fluid":
```

Comment:
Consider refactoring CreateSolverByParameters in _CreateALEFluidSolverByParameters, _CreateFluidSolverByParameters,

Reply:
I will do this if we add more ale-solvers